assets/src/edit-story/app/media/utils/useMediaUploadQueue.js

```diff
@@ -0,0 +1,356 @@
+/*
```

Comment:
This file is a little long. An chance of breaking into multiple files and it's own directory?